src/collapse/collapse.ts

```diff
@@ -0,0 +1,28 @@
+import {Directive, Input} from 'angular2/angular2';
+
+@Directive({
+  selector: '[ngb-collapse]',
+  host: {'class': 'collapse', '[class.in]': '_open', '[attr.aria-expanded]': '_open', '[style.height.px]': '_height'}
```

Comment:
Personally I would drop any code that deals with `height` for now, as we are not calculating / making any use of it for now.